### ISSUE TITLE
Breaking change in EFCore 8.0.0-preview.7.23375.4

### ISSUE THREAD

**Issue by Kiritsu** [KorzhCom/EasyData#170]
Hello,

I'm using EasyData.AspNetCore + EasyData.EntityFrameworkCore.Relational on version 1.4.18. 

I recently upgraded EFCore to 8.0.0-preview.7.23375.4, and since this preview version, there is a breaking change in their API which results in the following error when attempting to load the entities:

```
Method not found: 'System.Collections.Generic.IEnumerable`1<Microsoft.EntityFrameworkCore.Metadata.IProperty> Microsoft.EntityFrameworkCore.Metadata.IEntityType.GetProperties()'.
```

I tested that it still works on EFCore 8.0.0-preview.6.23329.4

**Comment by antifree** [comment]
Hello @Kiritsu!

Thank you for letting us know about the issue.

 We will add support of EF 8.0 once it will be final stable version.

**Comment by Alerinos** [comment]
@antifree I have the same problem, this change is permanent so you can make a fix.

**Comment by Alerinos** [comment]
@antifree In a few days release and it seems to me that the problem still occurs.

**Comment by korzh** [comment]
Hello,

We just have published version 1.4.19 of EasyData packages to NuGet. This update fully supports .NET 8.0
